src/CodeGenerator.php

```diff
@@ -367,6 +368,15 @@ private static function generateServices(
             $json = ResourcesGenerator::generateClientConfig($service, $gapicYamlConfig, $grpcServiceConfig);
             yield ["src/{$version}resources/{$service->clientConfigFilename}", $json];
             // TODO: Further files, as required.
+            // Resource: build_method.txt
+            $ctx = new SourceFileContext($service->gapicClientType->getNamespace(), $licenseYear);
+            $buildMethodFragments = BuildMethodFragmentGenerator::generate($ctx, $service);
+            foreach ($buildMethodFragments as [$fragmentName, $buildMethodFragments]) {
+                $buildMethodFragmentCode = BuildMethodFragmentGenerator::format(
+                    $buildMethodFragments->reduce('', fn ($v, $i) => $v . $i->toCode())
+                );
+                yield ["fragments/{$version}{$fragmentName}.build.txt", $buildMethodFragmentCode];
```

Comment:
```suggestion
                yield ["fragments/{$fragmentName}.build.txt", $buildMethodFragmentCode];
```

Paired with the other suggested change in BuildMethodFragmentGenerator